packages/wallets/src/features/cashier/screens/DepositLocked/DepositLocked.scss

```diff
@@ -0,0 +1,25 @@
+.wallets-deposit-locked {
+    margin-top: 9.2rem; //140px - 48px from cashier content
+
+    @include mobile {
+        margin-top: 1.6rem; //40px - 16px from cashier content
+    }
+}
+
+.wallets-link {
```

Comment:
that should be in WalletsLink link,
when I'm gonna want to make changes in WalletsLink styles component, I want to just look into WalletsLink.scss, I do not want to search for its styling across the codebase ;)
plus, worth double checking that this bit of styling won't have side effects,

Reply:
For this suggestion I'm not sure as wallets-link is being used somewhere else with different styling 🤔 Updated the implementation to use own classes instead to avoid styling conflicts 👍